features/support/world.ts

```diff
@@ -83,16 +85,20 @@ export class World {
     } else {
       const stdout = new PassThrough()
       const stderr = new PassThrough()
-      const cli = new Cli({
-        argv: args,
-        cwd,
-        stdout,
-        stderr,
-        env,
-      })
+      const environment: IRunEnvironment = { cwd, stdout, stderr, env }
+      const { options, configuration: argvConfiguration } =
```

Comment:
Looking at the 2 failing scenarios, I think this call to `parse` and the one below to `loadConfiguration` should also move inside of the try block.

Reply:
Moved inside try block